### DISCUSSION TITLE
Problem with showing transparent Glb

### DISCUSSION THREAD

**Discussion by ducle8110** [google/model-viewer#2443] [Q&A]
I created a transparent Glb body object in Blender,  in Blender the bone can be saw inside through the body but when i use that Glb in model-viewer the bone inside is not showing at all. 
When i use gltf viewer(https://gltf-viewer.donmccurdy.com/) it is showing well but i dont know how to use gltf-viewer.
Are there any setting to change may solve this?
Looking for help. Sorry for my bad English

Testing with
Windows 10
Browser: Chrome
using js: https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js

**Comment by elalish** [comment]
Please fill out the issue template completely instead of deleting it! I have no way to help you without your repro URL, or at least a link to your model.
